Document/0x05g-Testing-Network-Communication.md

```diff
@@ -169,6 +169,74 @@ OkHttpClient client = new OkHttpClient.Builder()
         .build();
 ```
 
+Applications that use WebView component may utilize event handler of the WebViewClient in order to perform some kind of "certificate pinning" on each request before that the target resource will be loaded. Verification code seems like the following:
```

Comment:
Applications that use **a** WebView component may utilize **the** event handler of the WebViewClient in order to perform some kind of "certificate pinning" on each request before that the target resource will be loaded. 

-> Please add the things I've put in bold
-> Please remove "that" in "that the target"

Instead of "Verification code seems like the following:" -> "The following code shows an example for verifying the Issuer DN of the certificate sent by the server. "